src/data/operations.js

```diff
@@ -144,20 +147,41 @@ export const operationsMap = [
     label: 'Begin Sponsoring Future Reserves',
     operationPane: BeginSponsoringFutureReserves,
     helpNote: 'Initiate a sponsorship. There must be a corresponding End Sponsoring Future Reserves operation in the same transaction.',
-    docsUrl: 'https://developers.stellar.org/api/resources/operations/object/begin-sponsoring-future-reserves',
+    docsUrl: 'https://developers.stellar.org/docs/start/list-of-operations/#begin-sponsoring-future-reserves',
   },
   {
     name: 'endSponsoringFutureReserves',
     label: 'End Sponsoring Future Reserves',
     operationPane: EndSponsoringFutureReserves,
     helpNote: 'End a sponsorship.',
-    docsUrl: 'https://developers.stellar.org/api/resources/operations/object/end-sponsoring-future-reserves',
+    docsUrl: 'https://developers.stellar.org/docs/start/list-of-operations/#end-sponsoring-future-reserves',
   },
   {
     name: 'revokeSponsorship',
     label: 'Revoke Sponsorship',
     operationPane: RevokeSponsorship,
     helpNote: 'Revoke sponsorship of a ledger entry.',
-    docsUrl: 'https://developers.stellar.org/api/resources/operations/object/revoke-sponsorship',
+    docsUrl: 'https://developers.stellar.org/docs/start/list-of-operations/#revoke-sponsorship',
+  },
+  {
+    name: 'clawback',
+    label: 'Clawback',
+    operationPane: Clawback,
+    helpNote: 'Creates a clawback operation.',
+    docsUrl: 'https://developers.stellar.org/docs/start/list-of-operations/#clawback',
+  },
+  {
+    name: 'clawbackClaimableBalance',
+    label: 'Clawback Claimable Balance',
+    operationPane: ClawbackClaimableBalance,
+    helpNote: 'Creates a clawback operation for a claimable balance.',
+    docsUrl: 'https://developers.stellar.org/docs/start/list-of-operations/#clawback-claimable-balance',
+  },
+  {
+    name: 'setTrustLineFlags',
+    label: 'Set Trust Line Flags',
+    operationPane: SetTrustLineFlags,
+    helpNote: 'Creates a trustline flag configuring operation.',
+    docsUrl: 'https://developers.stellar.org/docs/start/list-of-operations/#set-trustline-flags',
```

Comment:
Slight oddity I noticed due to the inconsistency here, so just adding a comment to get your thoughts: should the URL (and corresponding heading, obviously) be `#set-trust-line-flags` instead?

Reply:
The URL works for me with `#set-trustline-flags`. Is it changing?

Reply:
I'd vote for changing the link to keep them consistent.